### PR TITLE
Allow dragging a folder on the Welcome window to open a repository - Closes #468 and #247

### LINKED ISSUE #468: Drop Repos on Main Window
Lazy maybe, but I would like to be able to drag and drop a folder to open the repos.

### PR DESCRIPTION
**Pull request by sdrpa**

### CODE REVIEW

**Comment by sdrpa** [comment]
Found a typo. ```isHiglighted``` property in DragAndDropImageView.m should be corrected to ```isHighlighted```.

**Comment by hawkrives** [comment]
If it's a typo you introduced, why not just fix it in this one as another commit?

**Comment by sdrpa** [comment]
@hawkrives Done. I wasn't sure that new commit would become part of the pull request since I had already created the pull request. Thanks.

File: GitUp/Application/DragAndDropImageView.m [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  self.isHighlighted = NO;
+}
+
+- (void)drawRect:(NSRect)dirtyRect {
+  [super drawRect:dirtyRect];
+
+  if (self.isHighlighted) {
+    NSBezierPath *path = [NSBezierPath bezierPathWithRoundedRect:dirtyRect xRadius:10 yRadius:10];
+    // systemBlueColor from https://developer.apple.com/design/human-interface-guidelines/macos/visual-design/color/
+    NSColor *higlightColor = [NSColor colorWithRed:27.0/255.0
```
Comment (johankool) [inline GitUp/Application/DragAndDropImageView.m]: Typo in name, and also, perhaps better to use `[NSColor highlightColor]` or one of its siblings to have the color change appropriately with dark mode/user preferences?

**Comment by lucasderraugh** [comment]
@sdrpa So I've recently taken over as temporary maintainer of GitUp. If I can get an overview of visual changes here and squash changes into a single commit, I'd love to start the discussion on merging this in.

**Comment by lucasderraugh** [comment]
Going to close this out as we need updates to Dark Mode at this point. If somebody makes the changes and wants to reopen, I'll gladly look over it again.

**Pull request closed without merging**